v0/destinations/active_campaign/config.js

```diff
@@ -0,0 +1,31 @@
+const { getMappingConfig } = require("../../util");
+
+const BASE_ENDPOINT = "https://api.indicative.com/service";
+
+const CONFIG_CATEGORIES = {
+  IDENTIFY: { name: "ACIdentify",
```

Comment:
Can you please put the reference to the documentation pages here? It'll be helpful to understand the logic later